build/FileHelpers.cs

```diff
@@ -6,6 +6,20 @@ namespace Build
 {
     public static class FileHelpers
     {
+
+        public static void EnsureDirectoryExists(string dirPath)
+        {
+            if (!Directory.Exists(dirPath))
+            {
+                Directory.CreateDirectory(dirPath);
+            }
+        }
+
+        public static bool FileExists(string filePath)
+        {
+            return File.Exists(filePath);
+        }
```

Comment:
no biggie, but out of curiosity - does this add any benefit to just calling `File.Exists`?

Reply:
No benefit here, i had a bunch of code here to do some additional validation, that i removed but left the method

Reply:
have removed this